### ISSUE TITLE
Quirky Issues Training a Chinese Tortoise-TTS Model

### ISSUE THREAD

**Issue by Dinxin** [JarodMica/ai-voice-cloning#90]
Hello. I recently trained a tortoise-tts model using **300 hours** of Chinese speech data, with an average duration of **13 seconds** per sample. However, I encountered a very peculiar issue: the model's mel/text loss decreases normally on the training set, but sharply increases on the validation set. It seems that the model is overfitting.

Here are my parameter settings:

* Learning Rate = 0.0001
* Mel LR Ratio = 1
* Text LR Ratio = 1
* Learning Rate Scheme = Cosine Annealing
* Learning Rate Restarts = 4
* Batch Size = 128
* Gradient Accumulation Size = 1
* Validation Enabled = True (so I can observe the overfitting phenomenon).

The loss curves I observed are:
![image](https://github.com/JarodMica/ai-voice-cloning/assets/19572535/d095b687-a83f-4ed2-8dd7-b4abed0469d8)

When I reduced the learning rate to 5e-5, the problem seemed to be alleviated somewhat, but the trained model still lacks generalization ability on the validation set.
![image](https://github.com/JarodMica/ai-voice-cloning/assets/19572535/ecad4e64-c4a8-42e3-a223-501ce8a36c88)

As for the Chinese speech corpus, I used the G2Pw-pinyin module contained in Bert-VITS2 repo ([Extra-Fix branch](https://github.com/fishaudio/Bert-VITS2/tree/Extra-Fix)) to convert the Chinese characters into corresponding pinyin phonemes. Some data points look like:
![image](https://github.com/JarodMica/ai-voice-cloning/assets/19572535/8876ba47-482c-4a05-8223-cc4eef70378e)
